OutOfSchool/OutOfSchool.DataAccess/Enums/Subrole.cs

```diff
@@ -1,11 +1 @@
-﻿using System.Text.Json.Serialization;
-
-namespace OutOfSchool.Services.Enums;
-
-[JsonConverter(typeof(JsonStringEnumConverter))]
-public enum Subrole
-{
-    None,
-    ProviderDeputy,
-    ProviderAdmin,
-}
\ No newline at end of file
+﻿
```

Comment:
looks like file was not deleted

Reply:
Missed this issue while resolving conflicts. File deleted now